### ISSUE TITLE
Datepicker: Missing alt text on button on picklist

### ISSUE THREAD

**Issue by tmcconechy** [infor-design/enterprise#4318]
**Describe the bug**

**To Reproduce**
1. Go to http://localhost:4000/components/datepicker/example-index.html
2. Click on year/month to open the picker
3. Inspect the up and down arrow
4. See error - no alt text
5. Also can be seen in axe errors

**Expected behavior**

**Version**
- ids-enterprise: [4.31.1]

**Platform**
- Chrome

**Additional context**
https://jira.lawson.com/browse/LMCLIENT-31982

**Comment by jbrcna** [comment]
**QA Passed**
v4.33.0-dev
https://master-enterprise.demo.design.infor.com/components/datepicker/example-index.html?theme=uplift&variant=light&colors=0066D4
![image](https://user-images.githubusercontent.com/56722184/93335884-a4453c00-f859-11ea-8e0b-b9c75ecbe450.png)
